src/averageAges.js

```diff
@@ -14,8 +14,23 @@
  *
  * @return {number}
  */
+function sumAndRound(arrayOfReduce) {
+  const sumAges = arrayOfReduce.reduce((sum, age) => sum + age, 0);
+
+  return Math.round((sumAges / arrayOfReduce.length) * 100) / 100;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
+  const arrayPeople = century
+    ? people
+      .filter(item => (Math.ceil(item.died / 100)) === century)
+      .filter(({ sex }) => sex === 'm')
+      .map((item) => item.died - item.born)
```

Comment:
I think it's a bad idea to call a person an item)
Also, do you really need to call filter couple of times? Can't you do all the checks in one iteration?
And if you use map, then it's not arrayPeople anymore, but array of ages